getting-started/pages/register.tsx

```diff
@@ -0,0 +1,31 @@
+import { useRouter } from "next/router";
+import React from "react";
+import { RegisterDevice } from "../causal";
+import ClientOnly, { getOrGenDeviceId } from "../utils";
+
+export default function Page() {
+  return (
+    <ClientOnly>
```

Comment:
Is there a reason you just don't have the RegistrationPage component, and put the ClientOnly in there instead of the div?

Reply:
I'd have to wrap it all in an immediately executing function b/c I don't really want any part of the component executing on the server including the conditional router checks. I don't think that would look any better or be easier to read.